### DISCUSSION TITLE
INFO: EPanel 450 working!

### DISCUSSION THREAD

**Discussion by anthraxn8b** [fsaris/home-assistant-awox#109] [Q&A]
INFO: EPanel 450 working!

**Comment by fsaris** [comment]
Is it by any chance a Zigbee capable light?

**Comment by anthraxn8b** [comment]
@fsaris, 99% definitively not. It is a BT light and I have more or less no control over it. even the AWOX plugin hast no connection to it - just worked once. I would not recommend AWOX at all.
